unit-tests/src/test/scala/java/util/regex/PatternSuite.scala

```diff
@@ -372,6 +372,18 @@ object PatternSuite extends tests.Suite {
     syntax("foo(foo)bar)baz", "Missing parenthesis", 0)
   }
 
+  private val runStressTests = false
+  if (runStressTests) {
+    test("compile should not leak memory") {
+      println("===== AN ENDLESS STRESS TEST INVOKED =====")
+      while (true) {
+        for {
+          pattern <- (1 to 500).map(_.toString)
+        } Pattern.compile(pattern)
+      }
+    }
+  }
```

Comment:
If we can't run this test in CI, we should not include it as it's likely to get out of date in long-term.